README.md

```diff
@@ -3,11 +3,12 @@
 [![GitHub release (latest by date)](https://img.shields.io/github/v/release/AllenInstitute/MIES?style=plastic)](https://github.com/AllenInstitute/MIES/releases)
 [![Documentation](https://img.shields.io/badge/docs-doxygen%2Fbreathe%2Fsphinx-blue.svg?style=plastic)](https://alleninstitute.github.io/MIES/user.html)
 
-MIES is a proven \[[1](https://doi.org/10.1038/s41586-021-03813-8),
-[2](https://doi.org/10.1038/s41593-019-0417-0),
-[3](https://elifesciences.org/articles/37349),
-[4](https://doi.org/10.1038/s41586-021-03813-8),
-[5](https://doi.org/10.7554/eLife.65482)\] sweep based data acquisition
+MIES is a proven \[[1](https://www.science.org/doi/10.1126/science.abj5861),
```

Comment:
@t-b i think we can use the link below that provides full access. Can you verify that the link works for you?
https://www.science.org/stoken/author-tokens/ST-374/full

Reply:
Yes that link works for me. I can't save the PDF though due to DRM.

Reply:
@t-b yep, that's the expected behavior. Still, it's better than only being able to see the abstract.